validation/views.py

```diff
@@ -484,9 +492,99 @@ def register(request):
     return render(request, "validation/register.html", context)
 
 
+def view_issues(request):
+    issues = Issue.objects.filter(status=Issue.OPEN).order_by("id")
+    context = dict(
+        issues=issues,
+        auth=request.user.is_authenticated,
+        is_linguist=user_is_linguist(request.user),
+    )
+    return render(request, "validation/view_issues.html", context)
+
+
+def view_issue_detail(request, issue_id):
+    issue = Issue.objects.filter(id=issue_id).first()
+
+    form = None
+    if issue.recording:
+        form = EditIssueWithRecording(
+            request.POST, initial={"rec_id": str(issue.recording.id)}
+        )
+        rec = Recording.objects.filter(id=issue.recording.id).first()
+        phrase = None
+    if issue.phrase:
+        form = EditIssueWithPhrase(request.POST, initial={"phrase_id": issue.phrase.id})
+        phrase = Phrase.objects.filter(id=issue.phrase.id).first()
+        rec = None
+
+    if request.method == "POST":
+        if form.is_valid():
```

Comment:
Is there an `else` for this? i.e, when the `form` is **not** valid? Should this respond with a `4xx` error?

Reply:
Interesting. None of the other forms in `views.py` have an `else` clause to them, but maybe the should 🤔